### ISSUE TITLE
[RFC] IPA: allow switching off user private groups for trusted AD users

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#3178]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/3183
- Created at 2016-09-12 15:55:32 by [sbose](https://pagure.io/user/sbose)
- Assigned to nobody
---

IF algorithmic mapping is used trusted users from AD are assigned to a user private group (UPG). It should be possible to configure the id-range in a way to switch of UPGs and use the AD LDAP/PAC attribute primaryGroupID to determine the primary GID.

Related IPA ticket: https://fedorahosted.org/freeipa/ticket/6293

### Comments

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2016-09-12 17:43:23](https://pagure.io/SSSD/sssd/issue/3183#237184)**

Fields changed

cc:  => orion@cora.nwra.com

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2016-09-23 10:07:30](https://pagure.io/SSSD/sssd/issue/3183#237185)**

Unfortunately this patch depends on FreeIPA ticket #6293 which needs to be implemented first and is currently under "Future releases", so I'm moving this ticket to "Deferred" for the time being.

We can move this ticket back when FreeIPA implements https://fedorahosted.org/freeipa/ticket/6293

milestone: NEEDS_TRIAGE => SSSD Deferred

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2016-09-23 10:11:35](https://pagure.io/SSSD/sssd/issue/3183#237186)**

Fields changed

rhbz:  => todo

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2016-10-05 13:50:16](https://pagure.io/SSSD/sssd/issue/3183#237187)**

Not totally deferred, this ticket makes sense, it's "just" blocked by ​https://fedorahosted.org/freeipa/ticket/6293

milestone: SSSD Deferred => SSSD 1.16 beta

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2017-02-24 14:55:21](https://pagure.io/SSSD/sssd/issue/3183#237188)**

**Metadata Update from @sbose**:
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2020-03-13 11:31:07](https://pagure.io/SSSD/sssd/issue/3183#633032)**

**Metadata Update from @pbrezina**:
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field rhbz adjusted to https://bugzilla.redhat.com/show_bug.cgi?id=1649464 (was: todo)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2020-03-13 11:31:07](https://pagure.io/SSSD/sssd/issue/3183#633033)**

**Metadata Update from @thalman**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field rhbz adjusted to todo (was: https://bugzilla.redhat.com/show_bug.cgi?id=1649464)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue close_status updated to: None
- Issue tagged with: bugzilla

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2020-03-13 11:31:29](https://pagure.io/SSSD/sssd/issue/3183#633034)**

**Metadata Update from @pbrezina**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field rhbz adjusted to https://bugzilla.redhat.com/show_bug.cgi?id=1649464 (was: todo)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue **un**tagged with: bugzilla

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2020-03-13 11:31:47](https://pagure.io/SSSD/sssd/issue/3183#633035)**

**Metadata Update from @pbrezina**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue tagged with: bugzilla
